Authorship_Guidelines.md

```diff
@@ -1,10 +1,12 @@
 # Authorship Guidelines
 
-We adhere to the [ESA Code of Ethics on authorship](https://www.esa.org/esa/about/governance/esa-code-of-ethics/). Here we provide additional guidelines on authorship for the Beckman Research Group and collaborators. *This is a dynamic document; please suggest modifications to improve these general guidelines*. 
+We adhere to the [ESA Code of Ethics on authorship](https://www.esa.org/esa/about/governance/esa-code-of-ethics/). Below we provide additional guidelines on authorship for the Beckman Research Group and collaborators.These guidelines are to provide a basis for an open and continuing discussion on authorship among collaborators. They are inspired by [Oliver *et al.* (2018)](https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecs2.2206), other sources listed below, and past collaborative experiences. *This is a dynamic document; please suggest modifications to improve these general guidelines*. 
 
-Discussions on authorship is an interative process, and we encourage that they happen before initiation of a research project and continue throughout all stages of the project. These guidelines are to provide a basis for an open and continuing discussion on authorship among collaborators. They are inspired by [Oliver *et al.* (2018)](https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecs2.2206), other sources listed below, and past collaborative experiences. 
+We emphasize open and honest communication on authorship. Discussions on authorship is an interative process and norms vary by discipline, culture, and among research groups and are influenced by power differentials. Discussions on authorship should happen before initiation of a research project and continue throughout all stages of the project. The research team should come to a consensus as to what constitutes authorship contribution before the initiation of the research project, ensuring that every team member that contributes to work that culminates in a product receives co-authorship. Team members shoud discuss author responsibilities and criteria to assign author order. We will use [CRediT](http://credit.niso.org/contributor-roles-defined/) to describe each author's contributions.
```

Comment:
"Discussions...is" should be "Discussions...are"

Reply:
I think I did this! Thanks!